packages/editor/src/lib/editor/Editor.ts

```diff
@@ -79,6 +79,7 @@ import {
 import { EventEmitter } from 'eventemitter3'
 import { nanoid } from 'nanoid'
 import { EMPTY_ARRAY, atom, computed, transact } from 'signia'
+import { COLLABORATOR_TIMEOUT } from '../components/LiveCollaborators'
```

Comment:
Let's move this out into `constants.ts` if it's imported in more than one place.